deploy/operator.yaml

```diff
@@ -20,7 +20,7 @@ spec:
           command:
           - istio-operator
           - server
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
@ostromart I believe I like Always as a pull policy better.  I understand why this PR changes the default pull policy.  If Always is specified, docker will pull the image in all cases, even though it may not be rermotely pushed.

A sed operation on this line may make sense.  Do you have any thoughts on what is bettter?